x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/CohereService.java

```diff
@@ -71,7 +72,7 @@ public void parseRequestConfig(
                 taskSettingsMap,
                 serviceSettingsMap,
                 TaskType.unsupportedTaskTypeErrorMsg(taskType, NAME),
-                true
+                ConfigurationParseContext.REQUEST
```

Comment:
Switch to the enum since we're really trying to represent whether we're coming from a request or parsing a persistent configuration.